Networking/Networking/Remote/ProductCategoriesRemote.swift

```diff
@@ -89,6 +114,42 @@ public final class ProductCategoriesRemote: Remote {
         enqueue(request, mapper: mapper, completion: completion)
     }
 
+
+    /// Create new multiple `ProductCategory` entities.
+    ///
+    /// - Parameters:
+    ///     - siteID: Site for which we'll add new product categories.
+    ///     - names: Array of category names.
+    ///     - parentID: The ID for the parent of the categories.
+    ///     - completion: Closure to be executed upon completion.
+    ///
+    public func createProductCategories(for siteID: Int64,
+                                        names: [String],
+                                        parentID: Int64?,
+                                        completion: @escaping (Result<[ProductCategory], Error>) -> Void) {
+
+        let parameters = [
+            ParameterKey.create: names.compactMap { name in
```

Comment:
❓ Why do we need `compactMap` here when `names` is an array of non-optional strings?

Reply:
Good catch. Done in bb975e7973cfec8bcda5ce290b4594c0b49be372